package/MDAnalysis/analysis/rms.py

```diff
@@ -380,7 +380,10 @@ def __init__(self, atomgroup, reference=None, select='all',
              corresponding atom in `select`, and assumes ``None`` for `groupselections`.
 
         weights_groupselections : False or list of {"mass", ``None`` or array_like} (optional)
-             1. ``False`` will apply imposed weights to `groupselections` from ``weights`` option.
+             1. ``False`` will apply imposed weights to `groupselections` from
+             ``weights`` option if ``weights`` is either ``"mass"`` or ``None``. 
+             Otherwise will assume a list of length equal to length of 
+             `groupselections` filled with ``None`` values.
```

Comment:
This is surprising, could you show me where it does that? I'm not convinced this sentence is needed.

Reply:
I guess [this](https://github.com/z3y50n/mdanalysis/blob/0546385e68da94295e430a9e2c2fcc5ebc681ad8/package/MDAnalysis/analysis/rms.py#L579-L580) is the part. The otherwise part is referring to the `self.weights` not being "mass" or "None" (Basically when it is any iterable). Also this is the behavior you get from issue #2796.

Reply:
I see, thanks!